source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOAdaptiveCard.mm

```diff
@@ -62,7 +64,7 @@ + (ACOAdaptiveCardParseResult *)fromJson:(NSString *)payload;
     if (payload) {
         try {
             ACOAdaptiveCard *card = [[ACOAdaptiveCard alloc] init];
-            std::shared_ptr<ParseResult> parseResult = AdaptiveCard::DeserializeFromString(std::string([payload UTF8String]), std::string("1.5"));
+            std::shared_ptr<ParseResult> parseResult = AdaptiveCard::DeserializeFromString(std::string([payload UTF8String]), std::string(Version));
```

Comment:
please use plain c++ string since the line is using C++ interface and remove extern diff in header as it's not used anywhere else.
```C++
const std::string g_version = "1.6";
...
AdaptiveCard::DeserializeFromString(std::string([payload UTF8String]), g_version);